app/src/main/res/layout/activity_circuit_timer.xml

```diff
@@ -3,150 +3,151 @@
     xmlns:app="http://schemas.android.com/apk/res-auto">
 
     <RelativeLayout
+        android:id="@+id/main_layout"
         android:layout_width="match_parent"
         android:layout_height="match_parent">
 
         <ImageView
             android:id="@+id/close_button"
```

Comment:
This button is too close to the top of the screen. Needs to be moved down.

Reply:
fixed